web/ui/templates/service-discovery.html

```diff
@@ -11,6 +11,10 @@
     height: 65px;
     visibility: hidden;
   }
+
+  .label {
```

Comment:
Happy to move this out into the dedicated CSS pages if required, though I suspect then we'll want to scope this down to some service discovery class?

Reply:
It should be in the main CSS file, and as labels are displayed in many places we probably want this global.

Reply:
@brian-brazil Moved this into the `prometheus.css` file, I checked across the UI and it seems to work nicely.